code-challenges.test.js

```diff
@@ -23,11 +31,37 @@ const fibLength2 = 10
 
 
 // b) Create the function that makes the test pass.
+//create a function called Fibonacci
+//input: number greater than 2 
+//output: array that length containing the numbers of the Fibonacci sequence.
+// .length built in method or .map
+//convert number to array
+
+
+const fibonacci = (num) => {
+    let fibo = [0, 1]
+    for(let i = 2; i < num; i++) {
+        fibo.push(fibo[i-1] + fibo[i-2])
+    } return Array.from(fibo) + (num)
+}
+
+//Received: "0,1,1,2,3,56"
+//not sure where I am going wrong here? I get a very close output.
```

Comment:
You are super close on this!  Currently you are returning a string and not an array.  Since you are already pushing to an array called fibo, this is likely what you will want to return (no need to run additional methods on it).  Also, since the fibonacci sequence starts on 1, you may want to initialize your array to be [1, 1] and not [0,1]